eslint.config.ts

```diff
@@ -138,6 +140,7 @@ export default [
       'no-magic-numbers': 'off',
       'no-ternary': 'off',
       'no-undefined': 'off',
```

Comment:
Why?

Reply:
From https://typescript-eslint.io/rules/no-unused-vars/

```
export default tseslint.config({
  rules: {
    // Note: you must disable the base rule as it can report incorrect errors
    "no-unused-vars": "off",
    "@typescript-eslint/no-unused-vars": "error"
  }
});
```